File: sdkwork-ai-sdk-typescript/src/types/required-action.ts
import type { SubmitToolOutputs } from './submit-tool-outputs';

/** 所需操作 */
export interface RequiredAction {
  /** 操作类型 */
  type?: string;
  /** 提交工具输出 */
  submit_tool_outputs?: SubmitToolOutputs;
}

File: sdkwork-ai-sdk-java/src/main/java/com/sdkwork/ai/model/RequiredAction.java
package com.sdkwork.ai.model;

public class RequiredAction {
    private String type;
    private SubmitToolOutputs submitToolOutputs;

    public String getType() {
        return this.type;
    }
    
    public void setType(String type) {
        this.type = type;
    }

    public SubmitToolOutputs getSubmitToolOutputs() {
        return this.submitToolOutputs;
    }
    
    public void setSubmitToolOutputs(SubmitToolOutputs submitToolOutputs) {
        this.submitToolOutputs = submitToolOutputs;
    }
}
